projects/observability/src/shared/dashboard/widgets/topology/tooltip/topology-entity-tooltip.component.scss

```diff
@@ -7,7 +7,7 @@
   opacity: 0.92;
   box-shadow: 0px 4px 5px rgba(0, 0, 0, 0.2), 0px 3px 14px rgba(0, 0, 0, 0.12), 0px 8px 10px rgba(0, 0, 0, 0.14);
   border-radius: 4px;
-  max-width: 400px;
+  max-width: 800px;
```

Comment:
should we do this or wrap the text below instead of the ellipsis?

Reply:
this is fine :)